spell/spell-reviewer-mainnet-checklist.md

```diff
@@ -329,6 +329,7 @@ _Insert your local test logs here_
   * [ ] Exec Doc file name follows the format `Executive vote - Month DD, YYYY.md`
   * [ ] Extract _permanent_ URL to the raw markdown file and paste it below
     _Insert your Raw Exec Doc URL here_
+  * [ ] Ensure the URL uses commit hash that introduced last change to the Exec Doc, NOT merge commit (Note: correct commit hash can be found using `git log --pretty=oneline -1 -- "$LOCAL_PATH_TO_EXEC_DOC"` command executed inside [`makerdao/community` GitHub repo](https://github.com/makerdao/community))
```

Comment:
Nit: you can use the same exact formulation of the check in both checklist, no need to make it artificially different as discrepancies can create problems later on (eg unnecessary discussions)